consultation_analyser/support_console/file_models.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Literal
+
+from pydantic import BaseModel, computed_field, field_validator
+
+from consultation_analyser.consultations.models import ResponseAnnotation
+
+
+def read_from_s3(model, client, bucket: str, key: str):
+    s3_obj = client.get_object(Bucket=bucket, Key=key)
+    for line in s3_obj["Body"].iter_lines():
+        yield model.model_validate_json(line.decode("utf-8"))
+
+
+class DetailDetection(BaseModel):
+    themefinder_id: int
+    evidence_rich: Literal["YES", "NO"] = "NO"
+
+    @field_validator("evidence_rich", mode="before")
+    def coerce_null_evidence_rich(cls, v):
+        return "NO" if v is None else v
```

Comment:
this is the key change - if the value in the file is `null` it will be changed to "NO"